worlds/wargroove2/Locations.py

```diff
@@ -0,0 +1,120 @@
+from BaseClasses import Location
+
+location_table = {
+    "Humble Beginnings Rebirth: Talk to Nadia": 253001,
+    "Humble Beginnings Rebirth: Victory": 253002,
+    "Humble Beginnings Rebirth: Good Dog": 253003,
+    #########################################################
+    "Nuru's Vengeance: Victory": 253005,
+    "Nuru's Vengeance: Spearman Destroys the Gate": 253006,
+    "Nuru's Vengeance: Defeat all Dogs": 253007,
+    'Cherrystone Landing: Smacked a Trebuchet': 253008,
+    'Cherrystone Landing: Smacked a Fortified Village': 253009,
+    'Cherrystone Landing: Victory': 253010,
```

Comment:
Some of these use single quotes, some use double quotes, would recommend being consistent with one of the two.